### DISCUSSION TITLE
增大视频缓存时长

### DISCUSSION THREAD

**Discussion by I-Have-A-Long-Name-Just-Like-This** [the1812/Bilibili-Evolved#4642] [功能建议 / Ideas]
希望添加增大视频播放时缓存限制的功能，在以高倍速度(3倍以上)播放视频的时候视频缓存速度有时候跟不上播放速度，会卡顿

**Comment by Geeyun-JY3** [comment]
请善用搜索：https://github.com/the1812/Bilibili-Evolved/discussions/3564#discussioncomment-3388985
